libindy/src/services/pool/state_proof/node.rs

```diff
@@ -138,10 +138,21 @@ impl rlp::Decodable for Node {
     }
 }
 
-type NodeHash = generic_array::GenericArray<u8, <sha3::Sha3_256 as digest::FixedOutput>::OutputSize>;
+type NodeHash = sha3::digest::generic_array::GenericArray<u8, <sha3::Sha3_256 as sha3::digest::FixedOutput>::OutputSize>;
 pub type TrieDB<'a> = HashMap<NodeHash, &'a Node>;
 
 impl Node {
+    pub fn get_hash(&self) -> NodeHash{
```

Comment:
Should name of the function contains mentioning of RLP here?

Reply:
Maybe, but there are no other hashes so ...